framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -82,7 +82,6 @@ qx.Mixin.define("qx.ui.core.MExecutable",
      */
     _bindableProperties :
     [
-      "enabled",
```

Comment:
That's disabled. It's a bad idea to do so anyway because of various scopes. You can disable a Command without any problems. But this will not disable the Button.

The way it is designed, does not work: either the widget is ruling the enabled state, or the command. If the command is ruling, it breaks the ordinary disabled state and leaves the button usable, even it *looks* disabled.